### ISSUE TITLE
render _bundle error: TypeError: expected string or bytes-like object

### ISSUE THREAD

**Issue by abadger1** [django-webpack/django-webpack-loader#352]
{% render_bundle 'main' 'css' %}

error seen in ./manage.py runserver

 tags = utils.get_as_tags(
  File "/home/dave/skyscanner/venv/lib/python3.10/site-packages/webpack_loader/utils.py", line 71, in get_as_tags
    bundle = _get_bundle(loader, bundle_name, extension)
  File "/home/dave/skyscanner/venv/lib/python3.10/site-packages/webpack_loader/utils.py", line 47, in _get_bundle
    bundle = loader.get_bundle(bundle_name)
  File "/home/dave/skyscanner/venv/lib/python3.10/site-packages/webpack_loader/loader.py", line 116, in get_bundle
    filtered_chunks = self.filter_chunks(chunks)
  File "/home/dave/skyscanner/venv/lib/python3.10/site-packages/webpack_loader/loader.py", line 58, in filter_chunks
    ignore = any(regex.match(chunk)
  File "/home/dave/skyscanner/venv/lib/python3.10/site-packages/webpack_loader/loader.py", line 58, in <genexpr>
    ignore = any(regex.match(chunk)
TypeError: expected string or bytes-like object

venv) ➜  src git:(main) ✗ node -v
v14.21.3
(venv) ➜  src git:(main) ✗ npm -v
6.14.18
(venv) ➜  src git:(main) ✗ pip list | grep webpack
django-webpack-loader                   2.0.1
(venv) ➜  src git:(main) ✗ 

Django version 4.2.4, using settings 've.settings.development'

**Comment by abadger1** [comment]
npm run build appears to run fine - lost as to how to debug further

**Comment by abadger1** [comment]
https://stackoverflow.com/questions/73007144/django-webpack-loader-render-bundle-error

**Comment by abadger1** [comment]
![image](https://github.com/django-webpack/django-webpack-loader/assets/95862524/e0e685fb-f93f-4da5-9ec9-01d4206a3cb5)

**Comment by abadger1** [comment]
Have tried to upgrade all packages to node v18.17.1 compliant - still see same issue as above

**Comment by abadger1** [comment]
```
    tags = utils.get_as_tags(
  File "/home/dave/skyscanner/venv/lib/python3.10/site-packages/webpack_loader/utils.py", line 71, in get_as_tags
    bundle = _get_bundle(loader, bundle_name, extension)
  File "/home/dave/skyscanner/venv/lib/python3.10/site-packages/webpack_loader/utils.py", line 47, in _get_bundle
    bundle = loader.get_bundle(bundle_name)
  File "/home/dave/skyscanner/venv/lib/python3.10/site-packages/webpack_loader/loader.py", line 116, in get_bundle
    filtered_chunks = self.filter_chunks(chunks)
  File "/home/dave/skyscanner/venv/lib/python3.10/site-packages/webpack_loader/loader.py", line 58, in filter_chunks
    ignore = any(regex.match(chunk)
  File "/home/dave/skyscanner/venv/lib/python3.10/site-packages/webpack_loader/loader.py", line 58, in <genexpr>
    ignore = any(regex.match(chunk)
TypeError: expected string or bytes-like object
```

**Comment by AntonBox** [comment]
any progress?

**Comment by AntonBox** [comment]
@abadger1 the reason is because you use previously generated output file 'webpack-stats.json'. delete it!
and probably you have wrong STATS_FILE path. I guess adding 'static' to the path will help you.

**Comment by grimmq52156** [comment]
@abadger1, I just upgraded my legacy python project and encountered the exact same problem. The issue was that my package.json was using an old version of webpack-bundle-tracker, which was v0.7.0. After upgrading my webpack-bundle-tracker to v2.0.1 (the same as django-webpack-loader) and updating my webpack.config.js according to the latest 'configuring-webpack-bundle-tracker' section in the [readme](https://github.com/django-webpack/django-webpack-loader#configuring-webpack-bundle-tracker), I successfully solved the problem.
